lib/rollbax/item.ex

```diff
@@ -33,4 +38,28 @@ defmodule Rollbax.Item do
       Map.put(data, "custom", meta)
     end
   end
+
+  defp language do
+      "Elixir #{System.version}"
```

Comment:
Please use 2-space indentation and use concatenation `<>` operator instead of interpolation.